restapi/src/main/java/io/stargate/web/docsapi/resources/ReactiveDocumentResourceV2.java

```diff
@@ -164,6 +166,75 @@ public void postDoc(
                 asyncResponse, ErrorHandler.EXCEPTION_TO_RESPONSE));
   }
 
+  @POST
+  @ManagedAsync
+  @ApiOperation(
+      value = "Write multiple documents in one request",
+      notes =
+          "Auto-generates an ID for the newly created document if an `idPath` is not provided as a query parameter. When an `idPath` is provided, this operation is idempotent.",
+      code = 202)
+  @ApiResponses(
+      value = {
+        @ApiResponse(code = 202, message = "Accepted", response = MultiDocsResponse.class),
+        @ApiResponse(code = 400, message = "Bad request", response = ApiError.class),
+        @ApiResponse(code = 401, message = "Unauthorized", response = ApiError.class),
+        @ApiResponse(code = 403, message = "Forbidden", response = ApiError.class),
+        @ApiResponse(code = 500, message = "Internal Server Error", response = ApiError.class)
+      })
+  @Path("collections/{collection-id}/batch")
```

Comment:
What happens when a batch only partially succeeds? Should we mention that the updates to multiple docs is not atomic  or isolated?

Reply:
> This implementation actually fires a batch per document in-parallel, making each batch statement a single partition batch, thus more effective. However, this would mean that if one document write fails, other will succeed. My idea would be that the result of the HTTP call sends back the IDs of documents that were successfully written. Write errors are effectively ignored.

I see. You've answered this. We should mention these limitations in the docs.

Reply:
@mpenick @EricBorczuk I am not kinda worried about this not atomic batch.. Actually this means the `../batch` endpoint will always return `200`.. This seems wrong in a way and we would never get it in the metrics (OK logs would be there).. Maybe I should have keep the behavior until now and fire one big batch for all documents, which succeeds or not.. Not sure what to do, can we talk 5 mins about this please?

Reply:
I think it's fine too as long as its semantics are documented.